server/src/MessageHandler.ts

```diff
@@ -143,7 +143,7 @@ export function handleDefinitionRequest(
 
   const modelDefinition = 'model '
   // get start position of model type
-  const index = documentText.indexOf(modelDefinition + modelName + ' ')
+  const index = documentText.search(modelDefinition + modelName + /\s+/)
   const buf = documentText.slice(0, index)
   const EOL = '\n'
```

Comment:
Line endings on Windows are `\r\n` (and `\n` on macOS)? Maybe I'm just confused 🤔

Reply:
Not sure, but I replaced this part with a different function not needing the EOL definition.